Store.java

```diff
@@ -0,0 +1,128 @@
+import Java.util.ArrayList;
+import Java.util.Date;
+import Java.util.Calendar; 
+import Java.util.map;
+import Java.util.LinkedHashMap; 
+
+public class Store{
+
+	private int openingHour; 
+	private int closingHour;
+	private ArrayList<Product>  productList = new ArrayList<Product>();  
+	private double dailyProfit; 
```

Comment:
No need to keep track as a variable. You can easily calculate it whenever needed.